src/libstd/sys/unix/process/process_unix.rs

```diff
@@ -189,14 +189,18 @@ impl Command {
                 t!(cvt(libc::setgid(u as gid_t)));
             }
             if let Some(u) = self.get_uid() {
-                // When dropping privileges from root, the `setgroups` call
-                // will remove any extraneous groups. If we don't call this,
-                // then even though our uid has dropped, we may still have
-                // groups that enable us to do super-user things. This will
-                // fail if we aren't root, so don't bother checking the
-                // return value, this is just done as an optimistic
-                // privilege dropping function.
-                let _ = libc::setgroups(0, ptr::null());
+                //TODO: relibc and Redox kernel
+                #[cfg(not(target_os = "redox"))]
```

Comment:
Could this use `if cfg!` instead of `#[cfg]`?

Reply:
Yep